sublime-package.json

```diff
@@ -26,6 +26,41 @@
               "type": "string",
               "deprecationMessage": "Use the \"selector\" key instead."
             },
+            "FileWatcher": {
+              "type": "object",
+              "examples": [
+                {
+                  "glob": "{**/*.js,**/*.ts,**/*.json}",
+                  "kind": ["create", "change", "delete"],
+                  "ignores": ["**/.git/**", "**/node_modules/**", "**/.hg/**"]
```

Comment:
Might be good to revise the key naming here.
- `kind` - maybe it should be called `type`. Spec uses `kind` when registering but `type` when triggering events.
- `glob` and `ignores` - maybe `patternGlob` and `ignoresGlob` for consistency.

Reply:
I think `glob` and `ignores` is fine. Best to avoid snake_case vs camelCase by using a single noun ;) I have no real opinion about kind vs type

Reply:
If `ignores` is a glob and I'm not mentioning that in the key name then, for consistency, I think I'll rename `glob` to `pattern`. That will be more consistent.